Add IAG Summer School 2025 blog post (#62)
compgeolab/website: news/iag-summer-school-2025.md

---
title: "XXVII Geophysics Summer School"
date: 2025-03-18
author: Yago Moreira Castro
---

{%- import "macros.html" as macros %}

> From February 17 to 21, 2025, the **[XXVII Geophysics Summer School](https://www.iag.usp.br/eventos/curso-extensao-escola-verao-2025) at [IAG - USP](https://www.iag.usp.br/)** took place, offering an intense week of learning, knowledge exchange, and the development of essential skills for scientific research. Under the instruction of Professor **[Leonardo Uieda](https://www.leouieda.com/)**, participants had the opportunity to explore fundamental tools for a more efficient and reproducible scientific workflow, such as **Bash, Git, GNU Make, and LaTeX**.  

Reviewer: ```suggestion
From February 17 to 21, 2025, the **[XXVII Geophysics Summer School](https://www.iag.usp.br/eventos/curso-extensao-escola-verao-2025) at [IAG - USP](https://www.iag.usp.br/)** took place, offering an intense week of learning, knowledge exchange, and the development of essential skills for scientific research. Under the instruction of [Leo](../team#leouieda), [Arthur](../team#arthursmacedo), and [myself](../team#YagoMCastro), participants had the opportunity to explore some of the fundamental tools for efficient and reproducible scientific workflows: Bash, Git, GNU Make, and LaTeX.  
```

In addition to the technical content, the course provided a valuable space for networking and discussions, strengthening the academic community and fostering new collaborations. The participants' feedback was highly positive, highlighting the quality of the materials, the dedication of the team, and the interactive nature of the activities.  

> ### **The Experience as a Teaching Assistant**  

Reviewer: ```suggestion
## My experience as a teaching assistant
```

Os headings devem seguir a mesma ordem que usamos em artigos e teses. Um h3 só pode ser colocado dentro de um h2. Como não teve h2, aqui tem que ser um h2 (que está dentro de um h1 que é o título).

Serving as a teaching assistant in the course was both a challenging and rewarding experience. Supporting participants with their questions, assisting in practical activities, and contributing to the course organization provided a new perspective on the teaching and learning process. It was a valuable opportunity to develop pedagogical skills, work with diverse student profiles, and deepen my own knowledge in the field.  

We sincerely thank everyone who participated and contributed to the success of the event! Below is a special snapshot of our group during the course.  

{{ macros.figure("../images/news/kit.jpeg", caption = "XXVII Geophysics Summer School class for the course Survival Kit for Geoscientists", alt="A diverse group of individuals stands in a classroom setting, with computers and workstations in the foreground." ) }}

> ### **Workshop Repository**  

Reviewer: ```suggestion
## Workshop repository
```

> The materials and resources used during the workshop are available [here](https://github.com/compgeolab/kit).

Reviewer: ```suggestion
The materials and resources used during the workshop are available on the lab GitHub (in portuguese): [compgeolab/kit](https://github.com/compgeolab/kit) under open licenses. Feel free to reuse, share, and remix!
```
